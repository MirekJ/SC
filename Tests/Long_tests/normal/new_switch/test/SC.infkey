4b9d154a-cc74-9337-6c62-42b2ea3eed6a
