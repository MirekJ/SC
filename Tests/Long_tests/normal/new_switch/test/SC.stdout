Using RAN2 random generator

Patchy Spherocylinders version 3.6
-------------------------------------
Reading options...
 Pressure coupling type:                             1
 Pressure:                                           0.00000000
 Replica exchange pressure:                          0.00000000
 Average volume change attempts per sweep:           0.00000000
 Equilibration sweeps:                               0
 Sweeps between step size adjustments:               0
 Production sweeps:                                  30000000
 Sweeps between statistics samples:                  50000
 Sweeps between statistics reports:                  50000
 Average chain move attempts per sweep:              0.00000000
 Initial maximum displacement:                       0.21200000
 Inititial maximum angular change (degrees):         7.50000000
 Inititial maximum angular cone angle (degrees):     0.00000000
 Inititial maximum geo.box edge change:              0.00000000
 Initial maximum chain displacement:                 0.00000000
 Inititial maximum chain angular change (degrees):   0.00000000
 Temperature in kT/e:                                0.24426000
 Parallel tempering temperature in kT/e:             0.28426000
 Sweeps between replica exchange:                    0
 Sweeps between Grand-Canonical move:                0
 Sweeps between Cluster moves:                       0
 Wang-Landau method:                                 0 0
 Calculate the Wang-Landau method for atom type:     0
 Average type switch attempts per sweep:             0.10000000
 Number of Sweeps per pairlist update:               0
 Random number seed:                                 196150684
 Number of sweeps per writing out cluster info:      50000
 Sweeps between movie frames:                      50000

Reading topology...
Species:
Fields:11 0
Topology read of 1:        A (geotype: CPSC, 13) with parameters 1.33333 1.2 | 1.34695 0.3 | 60 5 | 0 
Fields:11 0
Topology read of 2:        B (geotype: CPSC, 13) with parameters 0.333333 1.2 | 1.34695 0.3 | 60 5 | 0 

Topology read for molecule: A 
particle 1: 	1 (with switchtype: 2 and delta_mu: 1.000000)
system:  A 164

Topology succesfully read. Generating pair interactions...
Generating GroupLists...

Reading configuration...
Box: (20, 20, 20)
Equilibration of maximum step sizes: 0 sweeps

Initializing energy functions...
Production run:  30000000 sweeps

sweep: 3000000 particles: 164 drift: -3.63798e-12
InteractionEnergy: -338.918, sweeps per hour: 3.95932e+06

sweep: 6000000 particles: 164 drift: 4.54747e-12
InteractionEnergy: -296.663, sweeps per hour: 3.97186e+06

sweep: 9000000 particles: 164 drift: 5.45697e-12
InteractionEnergy: -305.027, sweeps per hour: 3.96917e+06

sweep: 12000000 particles: 164 drift: 0
InteractionEnergy: -303.574, sweeps per hour: 3.97135e+06

sweep: 15000000 particles: 164 drift: 1.27329e-11
InteractionEnergy: -307.817, sweeps per hour: 3.97176e+06

sweep: 18000000 particles: 164 drift: -1
InteractionEnergy: -311.542, sweeps per hour: 3.96455e+06

sweep: 21000000 particles: 164 drift: 3.45608e-11
InteractionEnergy: -297.062, sweeps per hour: 3.96216e+06

sweep: 24000000 particles: 164 drift: -1
InteractionEnergy: -303.883, sweeps per hour: 3.97406e+06

sweep: 27000000 particles: 164 drift: 3.72893e-11
InteractionEnergy: -294.531, sweeps per hour: 3.97333e+06

sweep: 30000000 particles: 164 drift: 3.63798e-11
InteractionEnergy: -312.27, sweeps per hour: 3.97299e+06

Energy drift: 3.63798e-11 
Starting energy+pv: -6335.54617004 
Starting energy: -301.40108453 
Ending energy: -309.19991288 


******************************************************************************
*                               Moves Statistics                             *
******************************************************************************
Move                          Acc (%)   Rej (%)   Steps     
System:
A 164

Simulation time:2.7e+04 s
PairList generation time:0 %

DONE
Deallocating Conf...
Deallocating Sim...
Deallocating Topo...
